similar/main/digiobj.cpp

```diff
@@ -578,7 +578,7 @@ void digi_sync_sounds()
                                 &s.volume, &s.pan, s.max_distance );
 
 			} else if ( s.flags & SOF_LINK_TO_OBJ )	{
-				const auto objp = [&vcobjptr, &s]{
+				const auto objp = [&s]{
```

Comment:
NAK.  This line is required by <gcc-8.  If this change is merged, <gcc-8 fail to build this file.

I will push a configure test to suppress the clang warning in this case, since it actively encourages the reader to break other compilers.